tests/test_utilities.py

```diff
@@ -641,7 +629,6 @@ def test_send_email_notification_message_with_site_from_email_override(
             )
 
         enterprise_customer = EnterpriseCustomerFactory(site=site)
-        EnrollmentNotificationEmailTemplateFactory(enterprise_customer=None)
```

Comment:
I'm curious how we can just not make an email template here now and the test still work?

Reply:
Ah due to migration I think default self template already exists and is used

looks like this in test

```
(Pdb++) enterprise_template_config
<EnrollmentNotificationEmailTemplate (id: 1) Default template for type SELF_ENROLL>
```

Reply:
ahh right. I forgot tests here run the migrations!